suite-common/fiat-services/src/fetch.ts

```diff
@@ -0,0 +1 @@
+export const fetchUrl = (url: string) => fetch(url);
```

Comment:
nit: I feel like this is really redundant 😀

Reply:
I think it's OK, for me it's helping me to understand why it is happening. Also it's enforcing same type signature as native version where you can pass only url too.

Reply:
> I think it's OK, for me it's helping me to understand why it is happening
That is the thing why I did it this way...